eng/azure-pipelines.yml

```diff
@@ -90,7 +90,10 @@ stages:
       - job: SourceBuild_Managed
         displayName: Source-Build (Managed)
         pool:
-          vmImage: ubuntu-latest
+          ${{ if eq(variables.officialBuild, 'false' )}}:
+            vmImage: ubuntu-latest
+          ${{ if eq(variables.officialBuild, 'true' )}}:
+            name: NetCore1ESPool-Internal -equals Build.Ubuntu.1804.Amd64
```

Comment:
How does the `-equals` syntax work here? Is it supposed to have a `demands` section?

Reply:
Based on Matt's template I think this should have:
```yaml
demands: ImageOverride -equals Build.Ubuntu.1804.Amd64
```

Reply:
Ah, you're right, I misread that